ui/src/api/wfegraph.js

```diff
@@ -191,7 +191,7 @@ class Workflow2Graph {
         }
 
 
-    } else if(t1.type == 'DECISION'){
+    } else if(t1.type == 'DECISION' || t1.type == 'SWITCH'){
```

Comment:
Is a similar change needed for `INLINE` ?

Reply:
Do not see anything relevant to `LAMBDA` and so have not done anything special for `INLINE`.